Documentation/Troubleshooting/krew-plugin.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
In the `.pages` file in this directory, how about we add this page as the second one (after toolbox)?